src/utils/helpers/getPageElements.tsx

```diff
@@ -7,6 +7,7 @@ interface GetPageElementsProps {
 }
 
 const getPageElements = ({ elementsState, sectionIndex, pageNumber }: GetPageElementsProps) => {
+  // TODO: Should this also remove non-visible elements from the result?
```

Comment:
Yes, probably makes sense to filter them here.

Reply:
Cool. I'll work on that one now.

Reply:
Done! [869017f](https://github.com/openmsupply/application-manager-web-app/pull/218/commits/869017fb10ebc81f9b2f5f1fbaa2462645aa8ceb)